tools/release.sh

```diff
@@ -45,7 +45,8 @@ export GPG_TTY=$(tty)
 PORT=${PORT:-1234}
 
 # subpackages to be released
-SUBPKGS=${SUBPKGS:-"acme certbot-apache certbot-nginx"}
+SUBPKGS_NO_CERTBOT="acme certbot-apache certbot-nginx"
+SUBPKGS="certbot $SUBPKGS_NO_CERTBOT"
```

Comment:
I don't think it's necessary. It's nice in theory, but:

1. We've never used the feature. All of our packages are in lock step (e.g. Certbot x.y.z requires acme x.y.z) so releasing a subset of our packages doesn't make a lot of sense right now.
2. It isn't even respected by the current script in `master`. Certbot is always packaged, we expect there to be 12 lines of hash output, etc.

I think if we ever need this feature in the future, we can properly add it.